suggestions/templates/suggestions/problemsuggestion_form.html

```diff
@@ -119,7 +119,10 @@ <h1 class="alert-heading">Line length exceeded in solution!</h1>
         </td>
         <td>
           <button id="render-toggle" class="btn btn-info">Disable instant TeX</button>
-        </td>
+          {% if pk %}
+            <button id="delete" class="btn btn-warning" style="margin-left: 50px">Delete Suggestion
+            </button>
+          {% endif %}
```

Comment:
you dropped a `</td>`

Reply:
Why not just put the button in a different table `td`?

Reply:
That runs into an issue with the fact that `colspan=2` is set a bunch of times higher up, which would also need to be modified conditionally I think